index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
This is a suggestion.

Reply:
I don't think it saved whatever change in the text you made (be sure to click "Comment"). A suggestion might look like this: https://github.com/BrendanDang/Intro_Review_Pull_Requests/pull/1#discussion_r1090340530